doc/user/content/third-party/debezium.md

```diff
@@ -6,28 +6,25 @@ menu:
     parent: 'third-party'
 ---
 
-[Debezium](https://debezium.io/) works with Materialize in two capacities:
+You can use [Debezium](https://debezium.io/) to propagate Change Data Capture (CDC) data from a database to Materialize, for example MySQL or PostgreSQL.
 
-- Debezium adds change data capture (CDC) to MySQL or PostgreSQL databases,
-  which lets you propagate data from your database to Materialize
-- Debezium's CDC data (known as its envelope) is formatted in a way that
-  expresses arbitrary operations (e.g. updates and deletes) on the database's
-  data. This envelope is a powerful feature that lets Materialize perform more
-  complex analysis by understanding how upstream data is changing.
+Debezium emits records using an envelope that contains valuable information about the change captured, like the `before` and `after` values for each record. This envelope is a powerful structure that lets Materialize perform more complex analysis to understand all CRUD-like operations happening in the upstream database. For more details on CDC support in Materialize, check the [documentation](https://materialize.com/docs/sql/create-source/avro-kafka/#debezium-envelope-details).
```

Comment:
I think the term envelope is a little obscure, which was why it was parenthesized and de-emphasized previously. Maybe it warrants linking to a section on envelopes or having some other explanation?

Reply:
It definitely is, but IMO the previous description made it sound a bit more confusing than it really is. Adding "contains valuable information about the change captured, like the `before` and `after` values for each record" was my attempt at routing people to the right concept in their head, even if they don't know it as "envelope".

@ruf-io, how does this section sound to you?